packages/trader/src/Stores/Modules/Trading/Helpers/proposal.ts

```diff
@@ -103,6 +122,7 @@ export const getProposalInfo = (
         returns: `${returns.toFixed(2)}%`,
         stake,
         spot: proposal.spot,
+        validation_params: proposal?.validation_params,
```

Comment:
@kate-deriv how will this impact the proposal object we send to BE in production? can we please add test cases for this in proposal.spec.ts?🙏🏻

Reply:
It doesn't affect proposal request